hydra-node/test/Hydra/Chain/ZeroMQSpec.hs

```diff
@@ -5,12 +5,13 @@ module Hydra.Chain.ZeroMQSpec where
 import Cardano.Prelude
 import qualified Data.Set as Set
 import Data.String (String)
-import Hydra.Ledger (ParticipationToken (..))
-import Hydra.HeadLogic (OnChainTx (InitTx))
 import Hydra.Chain.ZeroMQ (catchUpTransactions, mockChainClient, runChainSync, startChain)
+import Hydra.HeadLogic (OnChainTx (InitTx))
+import Hydra.Ledger (ParticipationToken (..))
 import Hydra.Logging (nullTracer)
 import System.Timeout (timeout)
-import Test.Hspec (Spec, describe, it, shouldReturn)
+import Test.Hspec.Core.Spec (Spec, describe, it)
+import Test.Util (shouldReturn)
```

Comment:
Same remarks than for FireForgetSpec: We should reexport `Test.Hspec` from `Test.Util` and simplify imports for testing purpose